### ISSUE TITLE
Icon request: azure-devops

### ISSUE THREAD

**Issue by paulirwin** [FortAwesome/Font-Awesome#14058]
### What can it be used for?

Developer-facing applications that integrate with Microsoft's Azure DevOps (formerly Visual Studio Team Services, formerly Visual Studio Online, formerly Team Foundation Service, aka ADO, aka VSTS, aka VSO, aka TFS).
### Example image

![image](https://user-images.githubusercontent.com/1874103/46415699-cfc5cf00-c6f3-11e8-9bcc-2fc2bf1759ec.png)
Source: https://twitter.com/azuredevops

### Icon request checklist

- [X] This is a single icon or matched pair (Ex: `lock` / `unlock`)
- [X] The title starts with "Icon request: " and is followed by the requested icon name (Ex: `Icon request: magical-unicorn`)
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [X] I have included at least one example image if it could be helpful (optional)
- [X] The request is for a concrete object (it's harder to make an icon to represent happiness, it's easier to make a smiley face. ☺)

### Notes

This is a brand icon request, and would presumably need copyright/usage approval from Microsoft.

**Comment by mateohideg2** [comment]
Yes! I have the source code of a C# app than i made in Azure DevOps, and i want to put it icon with FontAwesome in the link to it.

**Comment by mateohideg2** [comment]
+1

**Comment by mateohideg2** [comment]
I have it in black
![azuredevops](https://user-images.githubusercontent.com/17548538/52172149-6e1c7c80-2748-11e9-814f-fb4319a84a9b.png)

**Comment by DaSchTour** [comment]
I hope when then also get **azure** as requested in #14072 and #5212 which sadly isn't event taged as brand icon request 😞

**Comment by qJake** [comment]
+1 This, along with the Azure logo, are very much needed by my org!

**Comment by AlthalusDGr8** [comment]
Guess this is still pending?

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.
